lib/exercism/deleted_iterations.rb

```diff
@@ -0,0 +1,10 @@
+class DeletedIterations < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :submission
+
+  def self.store_iterations(exercise, user_id)
+    exercise.submissions. each do |sub|
```

Comment:
looks like accidental whitespace between ```. each```.  Will run, but think best to just get rid of it